django/general/templates/general/cookies.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block main %}
+    <div class="container">
+        <div>
+            <h2>Cookies information</h2>
+            <p>
+                For information about cookies, including what they are and how you can manage them, please visit the University of Birmingham's <a href="https://www.birmingham.ac.uk/privacy/cookies.aspx">cookies policy page</a>.
+            </p>
+        </div>
+        <div>
+            <h2>Cookies used on this website</h2>
+
+            <table>
+                <tr>
+                    <th>Provider</th>
+                    <th>Cookie</th>
+                    <th>Purpose</th>
+                    <th>Expiry</th>
+                </tr>
+                <tr>
+                    <td>Google Analytics</td>
```

Comment:
Is this site have Google Analytics?

Reply:
it will, yeah, I'll set it up in a separate MR